src/mds/MDSRank.cc

```diff
@@ -3254,6 +3254,12 @@ void MDSRank::create_logger()
                             "exi", PerfCountersBuilder::PRIO_INTERESTING);
     mds_plb.add_u64_counter(l_mds_imported_inodes, "imported_inodes", "Imported inodes",
                             "imi", PerfCountersBuilder::PRIO_INTERESTING);
+    mds_plb.add_u64_counter(l_mdss_handle_client_caps, "handle_client_caps",
+                           "Client caps msg", "hcc", PerfCountersBuilder::PRIO_INTERESTING);
```

Comment:
add counter for both caps messages from client and caps message mds sends. also add counter for client caps messages with dirty caps and mds cap message that revoke caps.

Reply:
Done